<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>702</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Курсовая 1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1295</width>
     <height>650</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </item>
      <item>
       <widget class="QTableWidget" name="rootsTable">
        <property name="minimumSize">
         <size>
          <width>525</width>
          <height>0</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>Вещ. 1 корень</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Мним. 1 корень</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Выбор корня</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Вещ. 2 корень</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Мним. 2 корень</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelRe">
            <property name="text">
             <string>Вещественная часть</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputRe"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="labelIm">
            <property name="text">
             <string>Мнимая часть</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputIm"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="addRoots">
        <property name="text">
         <string>Добавить пару корней</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteRoots">
        <property name="text">
         <string>Удалить пару корней</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="notes"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QPushButton" name="calcWidthButton">
          <property name="text">
           <string>Посчитать центр и ширину</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calcButton">
          <property name="text">
           <string>Посчитать кэш функций</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="DrawingWidget" name="drawing" native="true">
        <property name="minimumSize">
         <size>
          <width>750</width>
          <height>500</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="tmin">
              <property name="text">
               <string>-10</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelt">
              <property name="text">
               <string>&lt; t &lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="tmax">
              <property name="text">
               <string>-10</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="labelDensity">
              <property name="text">
               <string>Мощность дробления</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="density">
              <property name="text">
               <string>1000</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="labelProdNumber">
              <property name="text">
               <string>Количество сомножителей</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="prodNumber">
              <property name="text">
               <string>100</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLineEdit" name="xmin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>-15</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelx">
              <property name="text">
               <string>&lt; x &lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="xmax">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>15</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLineEdit" name="ymin">
              <property name="text">
               <string>-10</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labely">
              <property name="text">
               <string>&lt; y &lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ymax">
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="drawButton">
        <property name="text">
         <string>Нарисовать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionInfo"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Отрисовка</string>
    </property>
    <addaction name="actionPlot_trig_polynome"/>
    <addaction name="actionPlot"/>
    <addaction name="actionPlot_2"/>
    <addaction name="separator"/>
    <addaction name="actionTrajectoryMode"/>
    <addaction name="actionFunctionMode"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Информация</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionPlot_trig_polynome">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Триг. многолчен</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>|триг. мног.|^2</string>
   </property>
  </action>
  <action name="actionPlot_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>φ̂</string>
   </property>
  </action>
  <action name="actionTrajectoryMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Траекторией</string>
   </property>
  </action>
  <action name="actionFunctionMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Функцией</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawingWidget</class>
   <extends>QWidget</extends>
   <header>drawingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
